### PR TITLE
Merge 7.11 changes to 7.12 branch

### PR DESCRIPTION
**Pull request by brokensound77**
# Issues
None

## Summary
There were changes made to the 7.11 branch for 7.11.2. This is to bring those changes over

### CODE REVIEW

File: rules/apm/apm_403_response_to_a_post.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 creation_date = "2020/02/18"
 maturity = "production"
-updated_date = "2020/10/26"
+updated_date = "2020/02/16"
```
Comment (bm11100) [inline rules/apm/apm_403_response_to_a_post.toml:4]:
```suggestion
updated_date = "2021/02/16"
```
Incorrect year for all modified rules.

**Review by rw-access — APPROVED** [review]

**Review by bm11100 — APPROVED** [review]
One comment on the years in the `updated_date`, other LGTM

File: rules/promotions/elastic_endpoint.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [metadata]
 creation_date = "2020/07/08"
 maturity = "production"
-updated_date = "2020/11/03"
+updated_date = "2021/02/16"
```
Comment (brokensound77) [inline rules/promotions/elastic_endpoint.toml:4]: The promotion rules were reverted but the updated_date remained which is fine

File: rules/apm/apm_403_response_to_a_post.toml (hunk 1 above)
Comment (brokensound77) [reply rules/apm/apm_403_response_to_a_post.toml]: pulled in the latest 7.11 updates which included the date fixes

**Comment by brokensound77** [comment]
Just verified that the reverted versions remained in tact 👍

**Pull request merged**